client/js/components/snippets/snippet.jsx

```diff
@@ -36,11 +40,13 @@ export default class Snippet extends React.Component {
         </Avatar>;
     return (
       <Card style={style}>
-        <CardHeader
-          style={{background: Colors.grey100}}
-          title={this.props.name || 'No title'}
-          subtitle="author link"
-          avatar={avatar} />
+        <div style={{display: 'inline-flex', background: Colors.grey100, width: '100%'}}>
```

Comment:
i added a div above, because adding just component RatingForm below CardHeader resulted in error (" Adjacent JSX elements must be wrapped in an enclosing tag " )